recipes-devtools/vpi/libnvvpi2_2.1.6.bb

```diff
@@ -42,12 +42,18 @@ do_install() {
     ln -snf lib/aarch64-linux-gnu ${D}/opt/nvidia/vpi2/lib64
 }
 
+pkg_postinst:${PN}() {
+    if [ -e "$D/opt/nvidia/vpi2/etc/ld.so.conf.d/vpi2.conf" ]; then
+        ln -sf /opt/nvidia/vpi2/etc/ld.so.conf.d/vpi2.conf $D${sysconfdir}/ld.so.conf.d/vpi2.conf
```

Comment:
You should probably also check if `$D${sysconfdir}/ld.so.conf.d` exists, and create it if needed.

Looking at the postinst scriptlet in the deb package from NVIDIA, it looks like it also copies the `pva_auth_allowlist` file to `${nonarch_base_libdir}/firmware`, and writes some values under `/sys/kernel/debug` to enable VPU app authentication... although checking those entries on my Xavier NX, they were already set to the correct value, so we can probably ignore that.

Reply:
Thanks @madisongh 
I will introduce new changes